File: AndroidJSONExample/app/src/main/java/com/example/shakil/androidjsonexample/Interface/RetrofitService.java
package com.example.shakil.androidjsonexample.Interface;

import com.example.shakil.androidjsonexample.Model.Movie;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface RetrofitService {

    @GET("marvel")
    Call<List<Movie>> getAllMovies();
}

File: KotlinJSONExample/app/src/main/java/com/example/shakil/kotlinjsonexample/Interface/RetrofitService.kt
package com.example.shakil.kotlinjsonexample.Interface

import com.example.shakil.kotlinjsonexample.Model.Movie
import retrofit2.Call
import retrofit2.http.GET

interface RetrofitService {
    @GET("marvel")
    fun getMovieList(): Call<MutableList<Movie>>
}